dipy/data/fetcher.py

```diff
@@ -611,33 +635,22 @@ def read_mni_template(contrast="T2"):
     >>> T2_nifti = read_mni_template("T2") # doctest: +SKIP
     Get both files in this order:
     >>> T1_nifti, T2_nifti = read_mni_template(["T1", "T2"]) # doctest: +SKIP
-
     """
-    folder = pjoin(dipy_home, 'mni_template')
+    files, folder = fetch_mni_template()
     file_dict = {"T1":pjoin(folder, 'mni_icbm152_t1_tal_nlin_asym_09a.nii'),
```

Comment:
Presumably the ROIs were made based on 'a'?

On Thu, Aug 6, 2015 at 11:03 AM, Jason D. Yeatman <notifications@github.com>
wrote:

> In dipy/data/fetcher.py
> <https://github.com/nipy/dipy/pull/681#discussion_r36445341>:
>
> >      """
> > -    folder = pjoin(dipy_home, 'mni_template')
> > +    files, folder = fetch_mni_template()
> >      file_dict = {"T1":pjoin(folder, 'mni_icbm152_t1_tal_nlin_asym_09a.nii'),
>
> I have no preference for a as oppose to c. If c is better, let's go with
> that
> … <#14f042e76c609d20_>
> On Thursday, August 6, 2015, Ariel Rokem <notifications@github.com>
> wrote: In dipy/data/fetcher.py <
> https://github.com/nipy/dipy/pull/681#discussion_r36440755>: > """ > -
> folder = pjoin(dipy_home, 'mni_template') > + files, folder =
> fetch_mni_template() > file_dict = {"T1":pjoin(folder,
> 'mni_icbm152_t1_tal_nlin_asym_09a.nii'), No rush. The ROIs that we are
> using for tractography segmentation are all based on the MNI template
> anyway. … <#14f040978a952b82_> On Thu, Aug 6, 2015 at 10:18 AM, Samuel
> St-Jean <notifications@github.com <javascript:_e(%7B%7D,'cvml','
> notifications@github.com');>> wrote: In dipy/data/fetcher.py <
> https://github.com/nipy/dipy/pull/681#discussion_r36440520>: > """ > -
> folder = pjoin(dipy_home, 'mni_template') > + files, folder =
> fetch_mni_template() > file_dict = {"T1":pjoin(folder,
> 'mni_icbm152_t1_tal_nlin_asym_09a.nii'), They also have a seperate license
> for commercial and non commercial usage, so maybe you can just point a
> download link for people while the re-licensing happens, so that people
> might have it in the meantime if it's that important. — Reply to this email
> directly or view it on GitHub <
> https://github.com/nipy/dipy/pull/681/files#r36440520>. — Reply to this
> email directly or view it on GitHub <
> https://github.com/nipy/dipy/pull/681/files#r36440755>.
> -- Jason D. Yeatman, PhD Assistant Professor, Institute for Learning &
> Brain Sciences (I-LABS) Department of Speech & Hearing Sciences University
> of Washington http://depts.washington.edu/bdelab/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/681/files#r36445341>.
>